.github/workflows/metadata_upload_manifest_dagger.yml

```diff
@@ -0,0 +1,22 @@
+name: Metadata Service CI
+
+on:
+  push:
+    branches:
+      - master
+jobs:
+  metadata_service_ci:
+    name: Upload modified metadata files to GCS
+    runs-on: medium-runner
+    steps:
+      - name: Checkout Airbyte
+        uses: actions/checkout@v2
+      - name: Run metadata upload pipeline
+        id: metadata-upload-pipeline
+        uses: ./.github/actions/run-dagger-pipeline
+        with:
+          subcommand: "metadata upload prod-airbyte-cloud-connector-metadata-service"
+          context: "master"
+        env:
+          CI_GITHUB_ACCESS_TOKEN: ${{ secrets.GH_PAT_MAINTENANCE_OCTAVIA }}
+          GCS_CREDENTIALS: ${{ secrets.METADATA_GCS_CREDENTIALS }}
```

Comment:
@bnchrch this secret does not exists, we need to create it. Do you already have a service account with R/W permissions to the bucket you want to write these files to?